sem_covid/entrypoints/etl_dags/eu_cellar_covid_worker.py

```diff
@@ -180,6 +181,39 @@ def upload_to_elastic_callable(**context):
     logger.info(f'Sent {json_file_name} file(s) to ElasticSearch.')
 
 
+def content_cleanup_callable(**context):
+    """
+
+    """
+    json_file_name = context['dag_run'].conf['filename']
+    logger.info(f'Cleaning up the the fragment {json_file_name}')
+    minio = StoreRegistry.minio_object_store(config.EU_CELLAR_BUCKET_NAME)
+    json_document = json.loads(minio.get_object(json_file_name).decode('utf-8'))
+
+    if json_document["content"]:
+        json_document["content"] = content_cleanup(json_document["content"])
+
+        minio.put_object(json_file_name, json.dumps(json_document))
+        logger.info(
+            f"Completed cleanup on {json_file_name} titled {json_document['title']} workID {json_document['work']}")
+    else:
+        logger.warning(
+            f"Skipping a fragment without content {json_file_name} titled {json_document['title']} workID {json_document['work']}")
+
+
+def content_cleanup(text: str) -> str:
```

Comment:
this cleanup has to be coordinated with the rest or "pre-processing" steps, also in ML DAGs. 
To me moved out into the " wrangling package.